omero/sysadmins/version-requirements.rst

```diff
@@ -233,16 +191,12 @@ Package lists
 
     * - Operating system
       - Details
-    * - CentOS 6 / RHEL 6
-      - EOL
     * - CentOS 7 / RHEL 7
       - `Reference <http://mirror.centos.org/centos/7/os/x86_64/Packages/>`__
+    * - Rocky 9
+      - `Reference <https://download.rockylinux.org/pub/rocky/9/BaseOS/x86_64/os/Packages/>`__
```

Comment:
Not sure this is appropriate, but is there some virtue in adding RHEL 9 package list ? (if it exists :)

Reply:
This follows what we have for other systems